.github/workflows/hydroshare-resouce-to-beta.yml

```diff
@@ -0,0 +1,59 @@
+name: MinIO Sync: Prod to Beta
+
+on:
+  workflow_dispatch:
+    inputs:
+      resource_ids:
+        description: 'Comma-separated list of resource IDs to check and sync'
+        required: true
+
+jobs:
+  check-and-sync:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Install MinIO CLI (mc)
+      run: |
+        curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
+        chmod +x mc
+        sudo mv mc /usr/local/bin/mc
+
+    - name: Set Up MinIO Aliases
+      env:
+        MINIO_PROD_URL: ${{ secrets.MINIO_PROD_URL }}
+        MINIO_BETA_URL: ${{ secrets.MINIO_BETA_URL }}
+        MINIO_ACCESS_KEY: ${{ secrets.MINIO_ACCESS_KEY }}
+        MINIO_SECRET_KEY: ${{ secrets.MINIO_SECRET_KEY }}
+      run: |
+        mc alias set prod $MINIO_PROD_URL $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+        mc alias set beta $MINIO_BETA_URL $MINIO_ACCESS_KEY $MINIO_SECRET_KEY
+
+    - name: Check and Sync Buckets for Resources
+      env:
+        RESOURCE_IDS: ${{ github.event.inputs.resource_ids }}
+      run: |
+        echo "Processing resource IDs: $RESOURCE_IDS"
+        IFS=',' read -ra RESOURCES <<< "$RESOURCE_IDS"
+
+        for RESOURCE_ID in "${RESOURCES[@]}"; do
+          echo "Checking resource: $RESOURCE_ID"
```

Comment:
You can use [mc mirror](https://min.io/docs/minio/linux/reference/minio-mc/mc-mirror.html) for the sync. By including the `--remove --overwrite` flags, it'll sync for the source to the target completely (including removing files that are on the target that aren't in the source).

You will also need to look up the bucket the resource is in, in order to sync the resource. There is an endpoint for this, GET `https://beta.hydroshare.org/hsapi/resource/$resource_id/quota_holder_bucket_name/`. An admin basic credentials are necessary to hit the endpoint.

You will also need to make sure the bucket exists on the target minio server. That can be done with [mc mb --ignore-existing](https://min.io/docs/minio/linux/reference/minio-mc/mc-mb.html#mc.mb.-ignore-existing)